spec/rubocop/cop/solidus/spree_discounted_amount_deprecated_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Solidus::SpreeDiscountedAmountDeprecated, :config do
+  let(:config) { RuboCop::Config.new('Solidus/SpreeDiscountedAmountDeprecated' => { 'Enabled' => true }) }
+
+  it 'registers an offense when using `#discounted_amount`' do
+    expect_offense(<<~RUBY)
+      discounted_amount
+      ^^^^^^^^^^^^^^^^^ Please use total_before_tax instead of discounted_amount.
+    RUBY
+  end
+
+  it 'does not register an offense when using `#total_before_tax`' do
+    expect_no_offenses(<<~RUBY)
+      total_before_tax
+    RUBY
+  end
```

Comment:
What is the purpose of these kind of tests?
I think it doesn't provide any benefits but just slow down that CI and makes the test suite hard to change.

Do you see my point? 

CC: @piyushswain , @abhishekgupta5 @MassimilianoLattanzio

Reply:
If this is the default behavior, I prefer to be consistent with rubocop guidelines

Reply:
Yea let's keep them. If they take too much time and make the CI slow, we can benchmark them and find why they are slow